src/components/PurchaseOrder/PO.js

```diff
@@ -204,6 +206,19 @@ class PO extends Component {
     this.unmountCloseOrderModal();
   };
 
+  approveOrder = async (e) => {
+    const { showToast, mutator, resources } = this.props;
+    const order = get(resources, ['order', 'records', 0]);
+    const orderNumber = get(order, 'poNumber', '');
+
+    try {
+      await updateOrderResource(order, mutator.order, { approved: true });
+      showToast('ui-orders.order.approved.success', 'success', { orderNumber });
+    } catch (error) {
+      await showUpdateOrderError(e, this.callout, this.openOrderErrorModalShow);
```

Comment:
Have you checked this working? I don't see `e` is defined here

Reply:
ok, it's defined but looks very confusing. Usually it's an error in catch block. Could you please give it a proper name?